packages/mobile/src/home/NotificationBox.tsx

```diff
@@ -229,37 +232,44 @@ export class NotificationBox extends React.Component<Props, State> {
         continue
       }
 
-      actions.push({
-        text: texts.body,
-        icon: notification.iconUrl ? { uri: notification.iconUrl } : undefined,
-        darkMode: notification.darkMode,
-        callToActions: [
-          {
-            text: texts.cta,
-            onPress: () => {
-              ValoraAnalytics.track(HomeEvents.notification_select, {
-                notificationType: NotificationBannerTypes.remote_notification,
-                selectedAction: NotificationBannerCTATypes.remote_notification_cta,
-                notificationId: id,
-              })
-              openUrl(notification.ctaUri, false, true)
-              trackRewardsScreenOpenEvent(notification.ctaUri, RewardsScreenOrigin.NotificationBox)
+      if (
```

Comment:
Before this line we already have some conditions of the format:

```
if (shouldSkipThisNotification) {
  continue
}
```
How about following that and doing the following:
```
if (!rewardsEnabled && notification.ctaUri.includes(Screens.ConsumerIncentivesHomeScreen)) {
  continue
}
```

This is a bit easier to read and the biggest advantage imo is that we avoid having a very big if statement that wraps almost 50 lines. This allows only having to think about the conditions in the beginning and not having to track braces.